test/discovery_failure_test.py

```diff
@@ -1,14 +1,26 @@
 import logging
 import os
+import subprocess
 import tempfile
 
-from testify import TestCase, assert_in, class_setup, class_teardown, run, test_discovery
-from testify.test_discovery import DiscoveryError
+import testify as T
 
 _log = logging.getLogger('testify')
 
 
-class BrokenImportTestCase(TestCase):
+HERE = os.path.dirname(os.path.abspath(__file__))
+
+
+def cmd_output(*cmd):
+    proc = subprocess.Popen(
+        cmd, cwd=HERE, stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+    )
+    stdout, stderr = proc.communicate()
+    assert proc.returncode != 0
```

Comment:
`assert ..., (stdout, stderr)` plz, so we have something to debug if it fails.

Imo assert should always have a second arg.